implementations/rust/ockam/ockam_api/src/cli_state.rs

```diff
@@ -750,8 +783,8 @@ impl NodeState {
         self.path.join("stderr.log")
     }
 
-    pub fn policies_storage_path(&self) -> PathBuf {
-        self.path.join("policies_storage.lmdb")
+    pub async fn policies_storage(&self) -> Result<LmdbStorage> {
+        Ok(LmdbStorage::new(self.path.join("authenticated_storage.lmdb")).await?)
```

Comment:
why `policies_storage.lmdb` becomes `authenticated_storage.lmdb`?

Reply:
Wow, that's a copy-paste error 🤦  Fixing it!